src/index.html

```diff
@@ -6,9 +6,65 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet"
+    href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img class="nav__logo"
+        alt="moyo_logo"
+        src="images/logo.png">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Put empty line in such cases for move readable
```suggestion
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#samsung" class="nav__link">
              samsung
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#smartphones" class="nav__link">
              smartphones
            </a>
          </li>
          
          <li class="nav__item">
```